src/api/methods.js

```diff
@@ -1,495 +1,511 @@
+    params: ["username", "password"]
+  },
+  {
+    api: "login_api",
+    method: "get_api_by_name",
+    params: ["database_api"]
+  },
+  {
+    api: "login_api",
+    method: "get_version"
+  },
+  {
+    api: "follow_api",
+    method: "get_followers",
+    params: ["following", "startFollower", "followType", "limit"]
+  },
+  {
+    api: "follow_api",
+    method: "get_following",
+    params: ["follower", "startFollowing", "followType", "limit"]
+  },
+  {
+    api: "follow_api",
+    method: "get_follow_count",
+    params: ["account"]
+  },
+  {
+    api: "follow_api",
+    method: "get_feed_entries",
+    params: ["account", "entryId", "limit"]
+  },
+  {
+    api: "follow_api",
+    method: "get_feed",
+    params: ["account", "entryId", "limit"]
+  },
+  {
+    api: "follow_api",
+    method: "get_blog_entries",
+    params: ["account", "entryId", "limit"]
+  },
+  {
+    api: "follow_api",
+    method: "get_blog",
+    params: ["account", "entryId", "limit"]
+  },
+  {
+    api: "follow_api",
+    method: "get_account_reputations",
+    params: ["lowerBoundName", "limit"]
+  },
+  {
+    api: "follow_api",
+    method: "get_reblogged_by",
+    params: ["author", "permlink"]
+  },
+  {
+    api: "follow_api",
+    method: "get_blog_authors",
+    params: ["blogAccount"]
+  },
+  {
+    api: "network_broadcast_api",
+    method: "broadcast_transaction",
+    params: ["trx"]
+  },
+  {
+    api: "network_broadcast_api",
+    method: "broadcast_transaction_with_callback",
+    params: ["confirmationCallback", "trx"]
+  },
+  {
+    api: "network_broadcast_api",
+    method: "broadcast_transaction_synchronous",
+    params: ["trx"]
+  },
+  {
+    api: "network_broadcast_api",
+    method: "broadcast_block",
+    params: ["b"]
+  },
+  {
+    api: "network_broadcast_api",
+    method: "set_max_block_age",
+    params: ["maxBlockAge"]
+  },
+  {
+    api: "market_history_api",
+    method: "get_ticker",
+    params: []
+  },
+  {
+    api: "market_history_api",
+    method: "get_volume",
+    params: []
+  },
+  {
+    api: "market_history_api",
+    method: "get_order_book",
+    method_name: "getMarketOrderBook",
+    params: ["limit"]
+  },
+  {
+    api: "market_history_api",
+    method: "get_trade_history",
+    params: ["start", "end", "limit"]
+  },
+  {
+    api: "market_history_api",
+    method: "get_recent_trades",
+    params: ["limit"]
+  },
+  {
+    api: "market_history_api",
+    method: "get_market_history",
+    params: ["bucket_seconds", "start", "end"]
+  },
+  {
+    api: "market_history_api",
+    method: "get_market_history_buckets",
+    params: []
+  },
+  // condenser_api is wrapper among others below three methods (sps_api,
+  // where "params" is object instead of array)
+  {
+    api: "condenser_api",
+    method: "find_proposals",
+    params: ["id_set"]
+  },
+  {
+    api: "condenser_api",
+    method: "list_proposals",
+    params: ["start", "order_by", "order_direction", "limit", "status"]
```

Comment:
transferring comment over-- limit should be second, here and in list_voter_proposals.  

(also first arg should probably be 'start' as well in list_voter_proposals, but maybe it's not important)

Also documented https://inertia186.github.io/devportal/apidefinitions/#database_api.list_proposals

(Probably already all known, but just in case not....)

Reply:
Thanks! I meant to fix these. Glad you caught it.